RNGNewAuraNotifier/Core/AppConstants.cs

```diff
@@ -20,7 +20,16 @@ internal static class AppConstants
     /// <summary>
     /// VRChatのデフォルトログディレクトリのパス
     /// </summary>
-    public static readonly string VRChatDefaultLogDirectory = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.UserProfile), "AppData", "LocalLow", "VRChat", "VRChat");
+#if DEBUG
+    public static readonly string VRChatLogDirectory = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.Desktop), "logtest");
+#else
+    public static readonly string VRChatLogDirectory = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.UserProfile), "AppData", "LocalLow", "VRChat", "VRChat");
+#endif
+
+    /// <summary>
+    /// アプリケーションの設定ディレクトリのパス
+    /// </summary>
+    public static readonly string ApplicationConfigDirectory = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData), GitHubRepoOwner, AppName);
```

Comment:
ここは、AppNameだとファイルパスとして考慮しないアプリケーション名が入ると思うので、不適切だと思います。
see #59

```suggestion
    public static readonly string ApplicationConfigDirectory = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData), GitHubRepoOwner, GitHubRepoName);
```
```suggestion
    public static readonly string ApplicationConfigDirectory = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData), GitHubRepoOwner, AppName);
```